### ISSUE TITLE
App not retrieving data

### ISSUE THREAD

**Issue by MohammedAlaaMorsi** [bufferapp/android-clean-architecture-boilerplate#54]
App can't load data from the URL , gives  404  https://joe-birch-dsdb.squarespace.com/s/team.json

**Comment by ghost** [comment]
Yes, the JSON file with Buffer team is not available anymore: https://github.com/bufferapp/android-clean-architecture-boilerplate/issues/17
